### PR TITLE
Enable IPv6 in CCI testsuite

### PR DESCRIPTION
**Pull request by spyrkob**
Missed the IPv6 option in initial migration

### CODE REVIEW

File: eap-job.sh [hunk 1]
```diff
@@ -216,6 +216,11 @@ else
     else
       export TESTSUITE_OPTS="${TESTSUITE_OPTS} -Djboss.dist=${WORKSPACE}/${TEST_JBOSS_DIST}"
     fi
+
+    if [ "$ip" == "ipv6" ];
```
Comment (rpelisse) [inline eap-job.sh:220]: "${ip}" ;) !

**Review by rpelisse — APPROVED** [review]
Fix the "$ip" in "${ip}" and merge!

File: eap-job.sh (hunk 1 above)
Comment (spyrkob) [reply eap-job.sh]: 🤦

**Comment by spyrkob** [comment]
thanks @rpelisse

**Pull request merged**